### PR TITLE
Allow empty SSID and password

### PR DESCRIPTION
**Pull request by lask**

### CODE REVIEW

File: vscode/src/utils.ts [hunk 1]
```diff
@@ -258,13 +258,13 @@ export async function promptForWiFiInfo(): Promise<WiFiInfo> {
     "prompt": "Enter Wi-Fi SSID"
   };
   const ssid = await Window.showInputBox(ssidPromptOptions);
-  if (!ssid) throw new Error("No Wi-Fi ssid provided");
+  if (ssid === undefined) throw new Error("No Wi-Fi ssid provided");
```
Comment (floitsch) [inline vscode/src/utils.ts:261]: I don't think an empty SSID is allowed.
Also: Generally it's "SSID".

**Review by floitsch — APPROVED** [review]
LGTM.

File: vscode/src/utils.ts (hunk 1 above)
Comment (lask) [reply vscode/src/utils.ts]: I think , the jury is still out on whether an empty SSID should be allowed.

Comment (lask) [reply vscode/src/utils.ts]: Fixed comment, reverted check.

**Comment by floitsch** [comment]
Did you test that having an empty string as password is what we need to do to connect to an open network?

**Pull request merged**